golem-debugging-service/test-components/shopping-cart/src/lib.rs

```diff
@@ -18,21 +16,6 @@ struct State {
     items: Vec<ProductItem>,
 }
 
-fn reserve_inventory() -> Result<(), &'static str> {
```

Comment:
nondeterminism was messing with being able to assert on the exact structure of logs emitted